docker/flower-waits-for-rabbitmq.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+until  [[ "$(curl -s -o /dev/null -L -w ''%{http_code}'' ${RABBITMQ_MNG}:15672)" != "200" ]]; do
+	  >&2 echo "Rabbitmq is unavailable - sleeping"
+	    sleep 3
+    done
```

Comment:
Unfortunately, this still does not work for me (even if you fix the condition to be `== "200"`). Here's what did work for me:
```shell
until curl --fail --silent --show-error http://cachito-api:8080/api/v1/status/short; do
    echo "Cachito is unavailable - sleeping"
    sleep 3
done
```
This (indirectly) makes you wait for:
- the Cachito API
- at least one Cachito worker
- the database
- rabbitmq
- nexus
- athens

We probably don't need to wait for *all* those things, but hey, at least it works (for me). I've also tried waiting for `cachito-api:8080/healtcheck` (only checks database), that does not seem to be good enough.

Reply:
@akhmanova, @athos-ribeiro can you check if this solution works for you locally?
@athos-ribeiro, would waiting for the `/status` endpoint be reasonable? Flower would refuse to start when there is anything wrong, even in unrelated services (nexus, athens). But this would only be an issue during deployments, and we usually expect things to be fine during those.

Reply:
I like the "wait for status endpoint solution". For the real production deployments, we can use an init container.

Reply:
@chmeliik Thank you. I changed it with your suggestion.

Reply:
This version works perfectly for me as well